demos/embeddings/deep-graph-infomax-cora.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>This isn't part of this PR directly (and so doesn't need to be fixed here), but I heard there were "standard" train/test/val splits for Cora? Do you know anything about this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
yeah I think that's what I used when I replicated the paper's results and I think its what most papers use. but I don't know (well haven't looked into) how/where to get these splits